src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/userpicker/userpicker.controller.js

```diff
@@ -1,8 +1,8 @@
 (function () {
     "use strict";
 
-    function UserPickerController($scope, usersResource, localizationService, eventsService) {
-        
+    function UserPickerController($scope, usersResource, entityResource, localizationService, eventsService) {
```

Comment:
@nul800sebastiaan can we remove `usersResource` then?

Reply:
Yep, done! 👍